src/Menu/Matcher/Voter/AdminVoter.php

```diff
@@ -47,11 +72,17 @@ public function matchItem(ItemInterface $item)
     {
         $admin = $item->getExtra('admin');
 
+        if (null !== $this->requestStack) {
+            $request = $this->requestStack->getMasterRequest();
```

Comment:
Master or current request should be used here?

Reply:
https://github.com/KnpLabs/KnpMenu/blob/master/src/Knp/Menu/Matcher/Voter/RouteVoter.php#L59 i guess it should be master request